arch/arm/Kconfig

```diff
@@ -82,6 +82,7 @@ config ARCH_CHIP_IMX6
 	select ARCH_CORTEXA9
 	select ARM_THUMB
 	select ARMV7A_HAVE_L2CC_PL310
+	select ARCH_HAVE_SETJMP if ARCH_TOOLCHAIN_GNU
```

Comment:
should we move the selection to ARCH_ARM?

Reply:
We can move it to ARCH_ARM after arm version added.